app/tests/unit/components/Form/ProjectQuarterlyReportForm.test.tsx

```diff
@@ -248,4 +248,62 @@ describe("The ProjectQuarterlyReportForm", () => {
       },
     });
   });
+
+  it("renders correct contract start date and TEIMP end date", () => {
```

Comment:
I added it because at one point in the refactor, I broke the start/end dates and none of the existing tests failed.  I can't remember exactly what I did, but I think I had the dates coming through as undefined (maybe a misplaced snake or camel case). However, now that you mention it, this test still uses dummy dates, so I don't think it's actually testing what I wanted. I'll look into adding an assertion to the cypress test. Thanks!